### DISCUSSION TITLE
new ada model! text-embedding-ada-002

### DISCUSSION THREAD

**Discussion by jan-casas** [nhaouari/obsidian-textgenerator-plugin#67] [Ideas]
Such a great plugin, extremely useful. 
could be posible to implement the new model (way faster, cheaper and better :))
Thank you so much.
https://openai.com/blog/new-and-improved-embedding-model/

**Comment by nhaouari** [comment]
This model is efficient for special tasks 
![image](https://user-images.githubusercontent.com/9850722/208734360-20f722db-63d0-45ce-92cc-c98820db8f9d.png)

One use case I think about it is to use it to discover new possible links with other notes.

**Reply by bSharpCyclist** [reply]
There is the AVA plugin, which has an option to find other related notes. Nice feature, would be cool if that was here!
